deploy/kubernetes/storageclass/ibmc-vpc-block-10iopsTier-StorageClass.yaml

```diff
@@ -6,7 +6,6 @@ provisioner: vpc.block.csi.ibm.io
 parameters:
   profile: "10iops-tier"            # The VPC Storage profile used.
   # iopsPerGB: "10"                 # The IOPS per Gigabyte that is supported for this profile. This is just for the user info.
```

Comment:
can we remove `iopsPerGB` as well and keep VPC URL in comment section on line `profile:` so that code editor will know at least or we can get the details what is supported and what not